### ISSUE TITLE
Boost.Python isn't installed currently

### ISSUE THREAD

**Issue by EinarElen** [LDMX-Software/docker#41]
**Describe the bug**
The Boost.Python library fails to be compiled and installed due to a configuration issue which seems to be caused by the bespoke build system failing to find python. 

**To Reproduce**
Look for the Boost.Python shared library files.

**What tag of the container are you using?**
v3.2

** Fixing 
To see the warning where it goes wrong, you can try running the same commands that we do when we build the container but from within the container (e.g. by doing `ldmx bash`). You can get it to work by modifying the "Jamfile" in the `libs/python/build/Jamfile' in the Boost source directory to specify `using python : 3.6 ;` instead of `using python ;`. That seems to be what the documentation suggests that you do, although you would think that there would be a flag of some kind somewhere to do the same thing. If there is, I haven't found one

**Comment by EinarElen** [comment]
The easiest solution would be to run something analogous to 
```bash
sed 's/using python ;/using python : 3.6 ;/' libs/python/build/Jamfile
```
before configuring. If anyone knows a better way, let me know

**Comment by EinarElen** [comment]
I've built a local container using 828fbe539947ec13948a0e7fcca976d57cb6c620 which installs the Boost.Python libraries correctly.

**Comment by EinarElen** [comment]
I've now also added `$LDMX_SW_INSTALL/lib` to `$PYTHONPATH` within the container which is required to import python bindings that are part of the shared objects we produce without a bunch of hassle

**Comment by tomeichlersmith** [comment]
Resolved with #46
